src/ui-shell/header/header-item.component.ts

```diff
@@ -25,7 +26,7 @@ import { Router } from "@angular/router";
 				[ngClass]="{'cds--header__menu-item--current' : isCurrentPage}"
 				[href]="href"
 				(click)="navigate($event)">
-				<ng-container *ngTemplateOutlet="content" />
+				<ng-content></ng-content>
```

Comment:
```suggestion
				<ng-container *ngTemplateOutlet="content" />
```

Undo, we shouldn't use ng-content twice in a condition.